internal/handlers/pg/msg_create.go

```diff
@@ -72,7 +72,16 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, lazyerrors.Error(err)
 	}
 
-	if err = h.pgPool.CreateTable(ctx, db, collection); err != nil {
+	if err := h.pgPool.CreateSettingsTable(ctx, db); err != nil && err != pgdb.ErrAlreadyExist {
+		return nil, lazyerrors.Error(err)
+	}
+
+	table, err := h.pgPool.GetTableName(ctx, db, collection)
+	if err != nil {
+		return nil, err
+	}
+
+	if err = h.pgPool.CreateTable(ctx, db, table); err != nil {
```

Comment:
Here you are trying to `CreateTable` but error message is with `collection` variable name. 
Could you please change this?

Reply:
It should be `collection` variable as this is error would be returned to the user and it would break integration test.

Reply:
> It should be `collection` variable as this is error that would be returned to the user and it would break integration test.

Can we add a comment here then? Thank you.

Reply:
Why? We return the wire protocol error here which is used by mongo clients. It should be enough. Although if someone would change `collection` to `table` here integration test would show it immediately.

Reply:
> Why? We return the wire protocol error here which is used by mongo clients. It should be enough. Although if someone would change `collection` to `table` here integration test would show it immediately.

let's make a call.